cloudhub/modules/ROOT/pages/vpc-tutorial.adoc

```diff
@@ -51,6 +51,10 @@ Don't associate Anypoint VPC with a design environment. You can deploy apps to t
 There is a cache timeframe between services that prevents rate limiting errors with different time configuration across Anypoint control planes.
 If you create an environment in Access Management and attempt to immediately use it to create or edit a VPC in Runtime Manager, you might experience a delay to see the newly created environment in the VPCs *Environments* dropdown.
 
+[NOTE]
+If you create a new VPC or update an existing one and associate an environment that is used in another VPC in the same region, any applications deployed in that other VPC are automatically deployed in the new or updated VPC as soon as it is restarted or updated.
```

Comment:
```suggestion
If you create a new VPC or update an existing one and associate an environment that is used in another VPC in the same region, any applications deployed in that other VPC are automatically deployed in the new or updated VPC instead as soon as it is restarted or updated.
```

Reply:
Also, to clarify, is the "it" in "as soon as it is restarted" referring to the VPC or the environment?

Reply:
If it's the environment, I recommend going with the longer suggestion from Slack:

If a new or updated VPC (VPC A) is associated with an environment already linked to another VPC (VPC B) in the same region, VPC B will lose the environment, and it is automatically attached to VPC A. As a result, any applications in that environment are deployed to VPC A when the environment is updated or restarted.